assets/eip-3475/utils/ownable.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
```

Comment:
```suggestion
// SPDX-License-Identifier: CC0-1.0
```

Reply:
its from the openzeppelin lib, i can mention that  along with this change